.github/workflows/ci-master-push.yaml

```
@@ -100,14 +99,14 @@ jobs:
       matrix:
         gochannel: ${{ fromJson(needs.go-channels.outputs.go-channels) }}
         test-case:
-          - { go-build-tags: snapd_debug,           snapd-debug: true,  go-test-race: false}
-          - { go-build-tags: withbootassetstesting, snapd-debug: false, go-test-race: false}
-          - { go-build-tags: nosecboot,             snapd-debug: false, go-test-race: false}
-          - { go-build-tags: faultinject,           snapd-debug: false, go-test-race: false}
-          - { go-build-tags: statelocktrace,        snapd-debug: false, go-test-race: false}
-          - { go-build-tags: snapdusergo,           snapd-debug: false, go-test-race: false}
-          - { go-build-tags: structuredlogging,     snapd-debug: false, go-test-race: false}
-          - { go-build-tags: "",                    snapd-debug: false, go-test-race: true }
+          - { go-build-tags: snapd_debug,           skip-coverage: false, snapd-debug: true,  go-test-race: false}
+          - { go-build-tags: withbootassetstesting, skip-coverage: false, snapd-debug: false, go-test-race: false}
+          - { go-build-tags: nosecboot,             skip-coverage: false, snapd-debug: false, go-test-race: false}
+          - { go-build-tags: faultinject,           skip-coverage: false, snapd-debug: false, go-test-race: false}
+          - { go-build-tags: statelocktrace,        skip-coverage: true,  snapd-debug: false, go-test-race: false}
+          - { go-build-tags: snapdusergo,           skip-coverage: false, snapd-debug: false, go-test-race: false}
+          - { go-build-tags: structuredlogging,     skip-coverage: true,  snapd-debug: false, go-test-race: false}
+          - { go-build-tags: "",                    skip-coverage: true,  snapd-debug: false, go-test-race: true }
```

we prob want coverage with the default one:
```suggestion
          - { go-build-tags: "",                    skip-coverage: false,  snapd-debug: false, go-test-race: true }
```

---

That's with the race flag. We do have coverage with the default one (in the step unit-tests). I'm happy to add coverage with the race flag though

---

I'm not sure why we have two jobs actually. It would be cleaner to just add another matrix entry that doesn't use the race flag and remove the other job

---

ah, sorry, missed that those are the 'special' tests